tests/test_click.py

```diff
@@ -11,6 +13,22 @@ def test_hello_world():
     assert "THM{" in result.output
 
 
+def test_filtration():
+    runner = CliRunner()
+    result = runner.invoke(main, ["--rarity", "0.5:", "--include_tags", "Identifiers,Media", "fixtures/file"])

```

Comment:
This test doesn't actually filter for me. 

```
python3 -m pywhat --rarity 0.1:0.2 --include_tags "Identifiers,Media" "fixtures/file"
```
![image](https://user-images.githubusercontent.com/10378052/120917485-7b92fa00-c6a7-11eb-86c8-357a346590f1.png)

Reply:
Tags do not also work from the CLI:
![image](https://user-images.githubusercontent.com/10378052/120917539-b5640080-c6a7-11eb-99df-67e3541dd5c1.png)

Reply:
Also, what's up with the double `file identified`?
![image](https://user-images.githubusercontent.com/10378052/120917580-e80df900-c6a7-11eb-8dc1-f6f0a0f9767f.png)

Reply:
Perhaps we should also write some tests to make sure this doesn't happen again, I.E. if `--rarity 0:0` or `--rarity -1:-1`  there should be no output? 
![image](https://user-images.githubusercontent.com/10378052/120917660-5783e880-c6a8-11eb-9d3b-732fe5631d5c.png)

Reply:
> What's up with the double `file identified`?
> ![image](https://user-images.githubusercontent.com/10378052/120917580-e80df900-c6a7-11eb-8dc1-f6f0a0f9767f.png)

IDK, it is present in the main branch, too